iphone/Classes/TiViewProxy.m

```diff
@@ -2935,13 +2935,44 @@ + (TiViewProxy *)unarchiveFromTemplate:(id)viewTemplate_ inContext:(id<TiEvaluat
   }
 
   if (viewTemplate.type != nil) {
-    TiViewProxy *proxy = [[self class] createProxy:viewTemplate.type withProperties:nil inContext:context];
-    [context.krollContext invokeBlockOnThread:^{
-      [context registerProxy:proxy];
-      [proxy rememberSelf];
-    }];
-    [proxy unarchiveFromTemplate:viewTemplate];
-    return proxy;
+    TiViewProxy *proxy = nil;
+    // First try a native proxy class...
+    @try {
```

Comment:
The approach I took in https://github.com/appcelerator/titanium_mobile/pull/9767 was to use a "module" tag that does class swizzling internally to construct classes from native modules. Obviously, this is not possible for common-js modules, but we could try to require the .js file manually and receive the proxy from there, similar to [here](https://github.com/appcelerator/titanium_mobile/blob/master/iphone/Classes/TiHost.m#L151). We could therefore add the Alloy-lookup into our require-scope temporary and keep it's logic hidden from the actual proxy. 

I agree that our core should **not** have any Alloy references as it becomes a huge mess across versions of both products over time, but I understand the state as a workaround. @vijaysingh-axway also spent some time on this already and also had issues with Alloy specifically and may be able to add something here as well.

Reply:
@hansemannn @vijaysingh-axway Well, this PR and the Android one are high priority due to customer request for this. It originated as an Alloy feature request and support in Alloy was added at the end of April...

Reply:
Can't a developer just use the widget view inside the list-view as a workaround? And are we planning to ship this in a 7.2.1, 7.3.0 or related? The Alloy PR should never been merged at this point, the SDK implementation was not in place.

Reply:
@sgtcoolguy @hansemannn  Isn't it good to use if-else instead of try-catch. e.g                                                                
if ([viewTemplate.type hasPrefix:@"Ti."] || [viewTemplate.type hasPrefix:@"Titanium.”]) {
//Native proxy
} else {
// Alloy widget
}

Reply:
@vijaysingh-axway Good idea! That's what also done inside the `createProxy:` factory method. By checking it before calling it, we also prevent a debug log that warns the user that the proxy cannot be constructed. See the change in my PR to this PR.